app/assets/javascripts/admin/controllers/admin-user-index.js.es6

```diff
@@ -41,6 +41,16 @@ export default Ember.Controller.extend(CanCheckEmails, {
     return userPath(`${username}/preferences`);
   },
 
+  @computed('model.second_factor_enabled')
+  show_second_factor(second_factor_enabled) {
+    return second_factor_enabled != null;
+  },
+
+  @computed('model.second_factor_enabled')
+  display_second_factor_enabled(second_factor_enabled) {
```

Comment:
I feel like this might be better put in the template, that way you don't have to disable escaping.

```hbs
{{#if model.second_factor_enabled}}
  {{i18n "yes_value"}}
{{else}}
  {{i18n "no_value"}}
{{/if}}
```